frontend/app/enrollment/components/selected-patient-view.tsx

```diff
@@ -0,0 +1,48 @@
+"use client"
+import { Card, CardContent, CardDescription, CardHeader, CardTitle } from '@/components/ui/card'
+import React, { useEffect, useState } from 'react'
+import useEnrollmentStore from '@/lib/store/enrollment-store'
+import Image from 'next/image'
+import { getBsn } from '@/lib/fhirUtils'
+
+export default function SelectedPatientView() {
+
+    const { patient } = useEnrollmentStore()
+    const bsn = getBsn(patient)
+
+    return (
+        <Card className='bg-primary text-gray-100'>
+            <CardHeader>
+                <CardTitle>Patient Information</CardTitle>
+            </CardHeader>
+            <CardContent>
+                <div className="grid grid-cols-4 gap-4">
+                    <div className="col-span-1 bg-gray-100 p-6 rounded-lg shadow-md flex items-center justify-center">
+                        <Image
+                            src="https://randomuser.me/api/portraits/men/75.jpg"
+                            width={144}
+                            height={144}
+                            alt="User Image"
+                            className="w-36 h-38 rounded-full mb-4"
+                        />
+                    </div>
+
+                    <div className="col-span-3 space-y-4">
+                        <div className="bg-gray-100 text-primary p-6 rounded-lg shadow-md">
+                            <h3 className="text-xl font-semibold">Name</h3>
+                            <p className="text-gray-600">{patient?.name?.[0].text || "Unknown"}</p>
```

Comment:
Names are very flexible, is this good enough (is `text` always present, at least in our cases for the first adopters?), or do we need to have more sophisicated rendering in future (if so, create issue for it?)

Reply:
Yeah, in the end these sort of methods should move to the `fhirUtils` and properly assemble the names and addresses etc, I'll add a TODO

Reply:
https://linear.app/zorg-bij-jou/issue/INT-228/frontend-improve-value-extraction-from-fhir-resoruces